src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -282,10 +283,11 @@ public function prepend($path, $data, $separator = PHP_EOL)
     /**
      * Append to a file.
      *
-     * @param  string  $path
-     * @param  string  $data
-     * @param  string  $separator
+     * @param  string $path
+     * @param  string $data
+     * @param  string $separator
      * @return bool
+     * @throws ContractFileNotFoundException
```

Comment:
This isn't the name of the Exception. The exception is `\Illuminate\Contracts\Filesystem\FileNotFoundException`